src/libcore/mem.rs

```diff
@@ -1148,10 +1148,19 @@ impl<T> MaybeUninit<T> {
     /// It is your responsibility to make sure `T` gets dropped if it got initialized.
     #[unstable(feature = "maybe_uninit", issue = "53491")]
     #[inline(always)]
-    pub const fn uninitialized() -> MaybeUninit<T> {
+    pub const fn uninit() -> MaybeUninit<T> {
         MaybeUninit { uninit: () }
     }
 
+    /// Deprecated before stabilization.
+    #[unstable(feature = "maybe_uninit", issue = "53491")]
+    #[inline(always)]
+    // FIXME: still used by stdsimd
```

Comment:
We should be able to future-deprecate it even though stdsimd uses it, right?

Reply:
We have `warn(deprecated_in_future)` and also on CI compiled with `-D warnings`. So I'm afraid I don't think we can.